src/context.ts

```diff
@@ -1,44 +1,37 @@
-import * as password from "./password";
-import * as wa_api from "./wild_apricot";
+import * as password from './password';
+import * as wa_api from './wild_apricot';
 
-
-export interface Logger
-{
+export interface Logger {
```

Comment:
My personal style is that curly brackets is K&R Style. Curlies should go on their own line for important structural things (interfaces, classes, non-anonymous functions), and on the same line for internal things (conditionals, loops, etc.)

Reply:
Didn't see any option for alternative bracketing styles unfortunately.